docs/blog/2019-11-10-how-to-convert-an-existing-gatsby-blog-to-use-mdx/index.md

```diff
@@ -0,0 +1,211 @@
+---
+title: "How to convert an existing Gatsby blog to use MDX"
+date: 2019-11-10
+author: Thomas Wang
+excerpt: "It can be a big tricky to add MDX to an existing blog. This blog post covers how to install and configure MDX to Gatsby's blog starter"
+tags:
+  - blogs
+  - mdx
+  - markdown
+---
+
+## What is MDX?
+
+MDX is a file format that lets you add JSX to Markdown files via a `.mdx` file! It was started by and is maintained by Gatsby's [John Otander](https://github.com/johno) and the [open source community on Github](https://github.com/mdx-js/mdx).
+
+With MDX, you can import React components and declare them alongside regular markdown syntax. This is useful to display specific component examples alongside the actual code, complex interactive components like charts, and reusable components across multiple pages like sign-up forms.
```

Comment:
Might want to add another catch-all to this last sentence. These are great use cases, but not the only ones!

Reply:
done